closure/goog/i18n/numberformat.js

```diff
@@ -261,7 +261,7 @@ goog.i18n.NumberFormat.prototype.setShowTrailingZeros =
 goog.i18n.NumberFormat.prototype.setBaseFormatting =
     function(baseFormattingNumber) {
   goog.asserts.assert(goog.isNull(baseFormattingNumber) ||
-      isFinite(baseFormattingNumber));
+          isFinite(baseFormattingNumber));
```

Comment:
You're correct. Will be part of the next commit. Wonder what I was reading from `gjshint` before.